### ISSUE TITLE
How to handle failure mode

### ISSUE THREAD

**Issue by solomatov** [soumith/ganhacks#40]
What's the best practice for handling failure mode, i.e. when the discriminator or the generator has 0 loss. Should we just terminate learning and debug the algorithm? Or should we rollback a couple of epochs back and continue training from there?

**Comment by John1231983** [comment]
@solomatov : Do you find the solution? I have same issue as you

**Comment by solomatov** [comment]
@John1231983 No, eventually I fixed the algorithm, but I don't remember what I did there.

**Comment by solomatov** [comment]
@John1231983 If you are interested, it's here: https://github.com/solomatov/cs231n-project
